Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+authors = ["MaidSafe Developers <dev@maidsafe.net>"]
+description = "Implementation of a DKG"
+documentation = ""
+edition = "2018"
+exclude = [""]
+homepage = "https://maidsafe.net"
+license = "GPL-3.0"
+name = "bls_dkg"
+readme = "README.md"
+repository = "https://github.com/maidsafe/BLS-DKG"
+version = "0.1.0"
+
+[dependencies]
+bincode = "1.1.4"
+failure = "~0.1.5"
+lazy_static = "~1.2.0"
+rand = "~0.4.2"
```

Comment:
Also use latest rand please.

Reply:
but the latest rand is at 0.7.3, and seems threshold_crypto depends on a bit old rand_core version.
So I'd suggest keep the RngAdapter, so that the latest rand can be used within this crate (avoid a potential adapter outside this crate)

Reply:
OK, I don't mind that. It's also worth keeping an eye on the threshold_crypto crate because it seems they already use rand 0.7.3 in master. So likely the next published version will have it too.